Sourabh_Kanwade/html/Assignment 5/A5_Home.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Home</title>
+        <link rel="stylesheet" href="./css/styles.css" />
+    </head>
+    <body>
+        <div id="background">
+            <h1>Sourabh Kanwade</h1>
+            <h2>About Me</h2>
+            <br />
```

Comment:
remove multiple br tags and use margin to give the space

Reply:
those `<br>` tags are required to work . I tried to give margin but it won't work as I wanted because I used `float:right`.So those `<br>` tags are essential.